aspnet/mvc/views/overview.rst

```diff
@@ -0,0 +1,91 @@
+Views Overview
+==============
+
+By `Steve Smith`_
+
+ASP.NET MVC Core controllers can return formatted results using *views*.
+
+.. contents:: Sections
+  :local:
+  :depth: 1
+
+What are Views?
+---------------
+
+In the Model-View-Controller (MVC) pattern, the *view* encapsulates the presentation details of the user's interaction with the app. Views are HTML templates with embedded code that generate content to send to the client. Views use :doc:`razor syntax <razor>`, which allows code to interact with HTML with minimal code or ceremony.
+
+ASP.NET Core MVC views are *.cshtml* files stored by default in a *Views* folder within the application. Typically, each controller will have its own folder, in which are views for specific controller actions. 
+
+.. image:: overview/_static/views_solution_explorer.png
+  :height: 100
+  :width: 200
+
+In addition to action-specific views, :doc:`partial views <partial>`, :doc:`layouts, and other special view files <layout>` can be used to help reduce repetition and allow for reuse within the app's views.
+
+Benefits of Using Views
+-----------------------
+
+Views provide `separation of concerns <http://deviq.com/separation-of-concerns/>`_ within an MVC app, encapsulating user interface level markup separately from business logic. ASP.NET MVC views use :doc:`razor syntax <razor>` to make switching between HTML markup and server side logic painless. Common, repetitive aspects of the app's user interface can easily be reused between views using :doc:`layout and shared directives <layout>` or :doc:`partial views <partial>`.
+
+:doc:`Tag helpers <tag-helpers/intro>` make it easy to add server-side behavior to existing HTML tags, avoiding the need to use custom code or helpers within views. Tag helpers are applied as attributes to HTML elements, which are ignored by editors that aren't familiar with them, allowing view markup to be edited and rendered in a variety of tools. Tag helpers have many uses, and in particular can make :doc:`working with forms <working-with-forms>` much easier.
```

Comment:
I think the last three paragraphs of this section should remain at the end of this article in their own section. This is too much detail to throw at the user at this point.